config/v1/types_infrastructure.go

```diff
@@ -477,6 +489,11 @@ type AWSPlatformStatus struct {
 	// +kubebuilder:validation:MaxItems=25
 	// +optional
 	ResourceTags []AWSResourceTag `json:"resourceTags,omitempty"`
+
+	// UserProvidedDNS indicates if the customer is providing their own DNS solution in place of the default
+	// provided by the underlying platform. Its value is set by the Installer and can have 2 possible
+	// values: "Enabled" and "Disabled" with "Disabled" being the default.
+	UserProvidedDNS UserProvidedDNS `json:"userProvidedDNS,omitempty"`
```

Comment:
Are you planning to actually default this value? If you aren't, then you don't need omitempty.

Given this is in the status and we expect this to never be Enabled by default, I think defaulting it could make sense here, WDYT?

Reply:
+1 for defaulting to `Disabled`. And agree that it is probably better to be in PlatformStatus than in PlatformSpec. Updating the enhancement to reflect that.